jcommune-view/jcommune-web-view/pom.xml

```diff
@@ -157,7 +157,7 @@
           <destinationFolder>${basedir}/src/main/webapp/resources/wro</destinationFolder>
           <cssDestinationFolder>${basedir}/src/main/webapp/resources/wro</cssDestinationFolder>
           <jsDestinationFolder>${basedir}/src/main/webapp/resources/wro</jsDestinationFolder>
-          <contextFolder>${basedir}/src/main/webapp/</contextFolder>
+          <contextFolder>${basedir}/src/main/webapp/,${basedir}\src\main\webapp\</contextFolder>
```

Comment:
Just one small thing - could copy-paste a part of your commit message as a comment to this line? People won't see the commit message unless they search for it.